lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/GroupCommunicationMessage.cs

```diff
@@ -46,7 +36,7 @@ private GroupCommunicationMessage()
             string source,
             string destination,
             T message)
-            : base(groupName, operatorName, source, destination)
+            : base(groupName, operatorName, source, destination, typeof(T))
```

Comment:
If base class accepts <T>, would it reach to the same purpose as base class keeps a type of T? I am not saying which one is better than the other :).